apps/devhub/tests/test_feeds.py

```diff
@@ -191,6 +191,17 @@ def test_rss_single(self):
         eq_(len(pq(r.content)('item')), 5)
         assert '<title>Recent Changes for %s</title>' % self.addon in r.content
 
+    def test_rss_unlisted_addon(self):
+        """Unlisted addon logs appear in the rss feed."""
+        self.addon.update(is_listed=False)
+        self.log_creates(5)
+
+        # This will give us a new RssKey
+        self.get_response(addon=self.addon.id)
+        key = RssKey.objects.get()
+        response = self.get_response(privaterss=key.key)
+        assert len(pq(response.content)('item')) == 5
+
```

Comment:
Can you also add XSS tests ? There are some below, just copy them with an unlisted addon.